data/migratedPages.yml

```diff
@@ -122,8 +122,11 @@ Filters:
 - filePath: "/docs/apis/plugintypes/filter/index.md"
   slug: "/docs/apis/plugintypes/filter/"
 Groups:
-- filePath: "/docs/guides/groups.md"
-  slug: "/docs/apis/subsystems/groups"
+- filePath: "/docs/guides/group/index.md"
+  slug: "/docs/apis/subsystems/group"
```

Comment:
Why "guides/groups.md" is defined here if it does not exists?

Reply:
Ah - that's a mistake. I moved it after original creation and missed the filePath update.